content/documents/api-from-scratch.mdx

```diff
@@ -0,0 +1,62 @@
+---
+type:           tutorial
+title:          Start .NET middleware from Scratch
```

Comment:
Rename your file to match the title as much as possible... something like:
`start-dotnet-middleware-from-scratch.mdx`